zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/validator/ElementRequestValidator.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.gateway.rest.validator;
+
+import static io.camunda.zeebe.gateway.rest.validator.ErrorMessages.ERROR_MESSAGE_EMPTY_ATTRIBUTE;
+import static io.camunda.zeebe.gateway.rest.validator.ErrorMessages.ERROR_MESSAGE_INVALID_ATTRIBUTE_VALUE;
+import static io.camunda.zeebe.gateway.rest.validator.RequestValidator.validate;
+
+import io.camunda.zeebe.gateway.protocol.rest.VariableRequest;
+import java.util.Optional;
+import org.springframework.http.ProblemDetail;
+
+public class ElementRequestValidator {
+
+  public static Optional<ProblemDetail> validateVariableRequest(
+      final VariableRequest variableRequest) {
+    return validate(
+        violations -> {
+          if (variableRequest.getVariables() == null || variableRequest.getVariables().isEmpty()) {
+            violations.add(ERROR_MESSAGE_EMPTY_ATTRIBUTE.formatted("variables"));
+          }
+          if (variableRequest.getOperationReference() != null
+              && variableRequest.getOperationReference() < 1) {
+            violations.add(
+                ERROR_MESSAGE_INVALID_ATTRIBUTE_VALUE.formatted(
+                    "operationReference", variableRequest.getOperationReference(), "> 0"));
+          }
```

Comment:
🔧 I guess we can extract this to the general request validator, passing in the operation reference value.

Reply:
Yes, I want to use a general approach for validate `operationReference` fields. I'll do it in a separate task if you don't mind